create-device-thinpool.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+    >&2 echo "Usage: create-devicemapper-thinpool.sh (docker|serviced) /dev/XXX [/dev/YYY...]"
```

Comment:
nitpick - the name of the file and filename in the Usage statement are different